### DISCUSSION TITLE
SQL parameter tracking

### DISCUSSION THREAD

**Discussion by MrJXL** [apache/skywalking#9255] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-booster-ui)

### What happened

When I configure the collection of SQL parameters, the query statements and parameters are displayed in the log tracking page，
version-9.1.0

### What you expected to happen

In the SQL trace log interface, SQL statements and parameters should not overlap

### How to reproduce

Open SQL parameter tracking. You can see that SQL statements and SQL parameters overlap in the MySQL log tracking interface

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Can't follow what you are saying. What do you mean `overlap`. Have to move to discussion.

**Reply by MrJXL** [reply]
![image](https://user-images.githubusercontent.com/17782002/174972615-c3fcd5ed-2fbe-4823-853a-6722817930ad.png)
version 9.1.0

**Reply by wu-sheng** [reply]
Oh, you mean this. This is a very small visualization issue. Would you fix this?

**Reply by wu-sheng** [reply]
I think we get a fix https://github.com/apache/skywalking-booster-ui/pull/113
